docs/source/admin/traffic_ops/using.rst

```diff
@@ -345,9 +347,10 @@ The fields in the Delivery Service view are:
+|                                              | See `regional geo-blocking <http://traffic-control-cdn.readthedocs.io/en/latest/admin/quick_howto/regionalgeo.html#configure-regional-geo-blocking-rgb>`_                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 +----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| IPv6 Routing Enabled                         | Default is yes, entering "No" allows you to turn off cdn response to IPv6 requests                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| IPv6 Routing Enabled                         | Default is "True", entering "False" allows you to turn off CDN response to IPv6 requests                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
It looks like the default for this is actually false

![image](https://user-images.githubusercontent.com/251272/39381994-411dd38c-4a21-11e8-9232-996dd1029ae7.png)

![image](https://user-images.githubusercontent.com/251272/39382015-51e8913e-4a21-11e8-96c6-eb3f9d3cb35b.png)